### DISCUSSION TITLE
URL as name instead of ip

### DISCUSSION THREAD

**Discussion by MIOsystems** [inventree/InvenTree#8136] [Q&A]
INVENTREE_SITE_URL is set to an ip address. All works well.

[192.168.1.10:8073/accounts/login](url)

But if i try to connect the server via the name resolve

[server:8073/accounts/login](url)

It displays an empty page. I assume the solution needs to be found in the reversed proxy server. But i have no clue how to set it up. Already tried to change the `INVENTREE_SITE_URL = https://server` with no result.

Any suggestions how to make this work?

**Comment by SchrodingersGat** [comment]
Have you configured the port as part of the `INVENTREE_SITE_URL` parameter?

**Comment by MIOsystems** [comment]
This is what i set in the environment setup:

```
INVENTREE_SITE_URL=http://192.168.1.10
INVENTREE_WEB_PORT=8073
INVENTREE_CORS_ORIGIN_ALLOW_ALL=true
```

**Comment by SchrodingersGat** [comment]
Please also provide some information about your setup. Are you running on docker (with Caddy) or the installer (with nginx)?

**Comment by MIOsystems** [comment]
Its a docker installation on a Synology server. So Container Manager is used. I added it as a Project (same as docker compose).
